Book-A-Desk.Api/BookingsHttpHandler.fs

```diff
@@ -65,6 +67,9 @@ module BookingsHttpHandler =
                             Date = booking.Date
                             User = { Email = booking.User.Email }
                         }
+                    sendEmailNotification booking
+                    |> ignore
```

Comment:
Why do you ignore? I honestly find it quite weird that we ignore what our own service returns. If we don't want it, simply make it so it doesn't return it :P

Reply:
As it is anyway the wrong place for a long term solution it was acceptable for me that if ever an exception happens that the API handles it as a server exception.  And of course in the real implementation we need a better handling :-)

Reply:
I'm not sure I understand? sendEmailNotification returns a unit, then we pass that to ignore, which just returns a unit again?

Reply:
I prefer 
do sendEmailNotification booking

Reply:
To Do: Removing ignore and raise any exception from this process.